docs/user-guide.rst

```diff
@@ -119,11 +136,17 @@ to ``False``. By default HTTP responses are closed after reading all bytes, this
 
 .. code-block:: python
 
-    >>> import io
-    >>> r = http.request('GET', 'https://example.com', preload_content=False)
-    >>> r.auto_close = False
-    >>> for line in io.TextIOWrapper(r):
-    >>>     print(line)
+    import io
+    import urllib3
+
+    http = urllib3.PoolManager() # Creating a seprate PoolManager instance.
+
+    r = http.request('GET', 'https://example.com', preload_content=False)
+    r.auto_close = False
+
+    for line in io.TextIOWrapper(r):
+        print(line)
+    #<!doctype html> ...
```

Comment:
Let's remove the `# ...` here only because it's kinda clunky since we're in a loop.

Reply:
Done.